apps/tangle-dapp/components/Lists/AssetList.tsx

```diff
@@ -75,20 +76,21 @@ export const AssetList = ({
                 onSelectAsset(asset);
                 onClose?.();
               }}
-              className="cursor-pointer w-full flex items-center gap-4 justify-between max-w-full min-h-[60px] py-[12px]"
+              className="cursor-pointer w-full flex items-center gap-4 justify-between max-w-full min-h-[60px] py-[12px] px-6"
             >
               <div className="flex items-center gap-2">
                 <TokenIcon
-                  size="lg"
+                  size="xl"
                   name={asset.symbol}
-                  className="mr-2 w-[38px] h-[38px]"
+                  className="mr-2"
+                  spinnersize="lg"
                 />
 
                 <div className="flex flex-col gap-1">
                   <Typography
                     variant="h5"
                     fw="bold"
-                    className="block cursor-default text-mono-200 dark:text-mono-0"
+                    className="cursor-default text-mono-200 dark:text-mono-0 flex items-center gap-1"
```

Comment:
Why flex and gap here? Is `asset.symbol` composed of various parts or components?